scrapers/data18.yml

```diff
@@ -1,35 +1,31 @@
 name: data18
+movieByURL:
+  - action: scrapeXPath
+    url:
+    - data18.com/movies
+    scraper: movieScraper
 sceneByURL:
   - action: scrapeXPath
     url:
-      - data18.com/content
+      - data18.com/scenes
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     common:
-      $performer: //p[b[text()='Starring:']]/a
-      $studio: //p[b[text()='Site:' or text()='Studio:']]/a
+      $performer: //b[text()='Pornstars / Cast']/../..//a[@class='bold gen']
+      $studio: (//b[text()="Studio"] | //b[text()="Network"])/../a[last()]
+      $movie: //b[text()="Movie:"]/following-sibling::a[1]
     scene:
-      Title: //h1/text()
+      Title: //span/following-sibling::h1/a/text()
       Date:
-        selector: //span[contains(text(),"Release date:") and not(contains(text(),"No date release yet"))]
-        postProcess:
-          - replace:
-              - regex: "^Release date:|,"
-                with:
-              # When day of month is not specified, the scraper defaults to <Month> 1st, <Year>.
-              # Comment the next two lines if you want to skip partial dates instead.
-              - regex: ([A-Za-z]+) (\d{4})
-                with: "$1 1 $2"
-          - parseDate: January 2 2006
+        selector: //time/@datetime
       Details:
-        selector: //p[b[text()='Story:']]
+        selector: //div[b[text()="Story"]] |  //div[b[text()="Story"]]/a | //b[contains(text(),"Movie Description")]/../text()
```

Comment:
`//div[b[text()="Story"]] |  //div[b[text()="Story"]]/a`

the second part `//div[b[text()="Story"]]/a` is not needed as stash will get all the inner text from the whole div
it even appends the  a (usually performers) at the end of the description due to the concat, so it should be removed